src/components/SelectChildForClass.tsx

```diff
@@ -28,6 +36,7 @@ export default function SelectChildForClass(
                     </b>
```

Comment:
^^ i marked this area bc currently the title of the class is hardcoded, i could pass in the class name as a param and fix it in this ticket if we want (not sure if we have an existing ticket for it, if we do i couldn't find it)

Reply:
lines 34-35^

Reply:
Yea good catch! Please patch this up for the team 🙏